ca/config.go

```diff
@@ -87,6 +89,69 @@ type CertificateUpdate struct {
 	Err  error
 }
 
+// ParsedJoinToken is the data from a join token, once parsed
+type ParsedJoinToken struct {
+	Version    int
+	RootDigest digest.Digest
+	Secret     string
```

Comment:
Not sure if it makes sense to add a comment to clarify each field, but I'll let you decide.